dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/ConnectionManagerServiceImpl.java

```diff
@@ -538,6 +542,9 @@ private J2CGlobalConfigProperties processServerPoolManagerProperties(AbstractCon
             if (pm.gConfigProps.getAgedTimeout() != agedTimeout)
                 pm.gConfigProps.setAgedTimeout(agedTimeout);
 
+            if (pm.gConfigProps.getAbortLongRunningInuseConnections() != abortLongRunningInuseConnections)
+                pm.gConfigProps.setAbortLongRunningInuseConnections(abortLongRunningInuseConnections);
```

Comment:
Be careful with this extra line between if statement and set.  Without {} you may call `setAbortLongRunningInuseConnections` regardless of the condition.